src/components/ha-combo-box.ts

```diff
@@ -246,8 +246,8 @@ export class HaComboBox extends LitElement {
     // delay this so we can handle click event for toggle button before setting _opened
     setTimeout(() => {
```

Comment:
Did you check if this was still needed?

Reply:
Yes, it's still needed to avoid flickering when selecting the item and to have the "create helper" working in the target picker